util-hadoop/src/test/java/com/google/cloud/hadoop/util/HadoopCredentialConfigurationTest.java

```
@@ -132,6 +133,7 @@ public void exceptionIsThrownForNoCredentialOptions() {
   }
 
   @Test
+  @Ignore
```

Probably we should not do this, because it can hide failures in the future.